src/main/java/club/sk1er/mods/autogg/AutoGG.java

```diff
@@ -20,271 +20,69 @@
 
 import club.sk1er.mods.autogg.command.AutoGGCommand;
 import club.sk1er.mods.autogg.config.AutoGGConfig;
-import club.sk1er.mods.autogg.listener.AutoGGListener;
-import club.sk1er.mods.core.universal.ChatColor;
-import com.google.gson.*;
-import com.google.gson.reflect.TypeToken;
-import net.minecraft.client.Minecraft;
-import net.minecraft.client.multiplayer.ServerData;
-import net.minecraftforge.client.ClientCommandHandler;
+import club.sk1er.mods.autogg.handlers.gg.AutoGGHandler;
+import club.sk1er.mods.autogg.tasks.RetrieveTriggersTask;
+import club.sk1er.mods.autogg.tasks.data.TriggersSchema;
 import net.minecraftforge.common.MinecraftForge;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.common.event.FMLInitializationEvent;
-import net.modcore.api.ModCoreAPI;
 import net.modcore.api.utils.Multithreading;
-import org.apache.commons.io.IOUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.lang.reflect.Array;
-import java.net.HttpURLConnection;
-import java.net.URL;
-import java.nio.charset.Charset;
-import java.util.*;
-import java.util.regex.Pattern;
+/**
+ * Contains the main class for AutoGG which handles trigger schema setting/getting and the main initialization code.
+ *
+ * @author ChachyDev
+ */
 
 @Mod(modid = "autogg", name = "AutoGG", version = AutoGG.VERSION)
 public class AutoGG {
-    @Mod.Instance("autogg")
-    public static AutoGG instance;
-
-    public static final String VERSION = "4.1";
-    private static final String[] ACCEPTED_CONFIG_VERSIONS = {"2"};
-    public static boolean validConfigVersion, triggerFetchSuccess = true; // independent of config
-    private final Logger logger = LogManager.getLogger("AutoGG");
-    private final String prefix = ChatColor.BLUE + "[AutoGG] " + ChatColor.RESET;
-    private static JsonObject triggerJson;
-    public static Map<String, String> triggerMeta;
+    public static final String VERSION = "4.2";
```

Comment:
This should be a placeholder that is replaced at build time

Reply:
Aren't replacing tokens not recommended at build time

Reply:
it can be inlined since it was only used for debug purposes when actually used